data/src/main/java/com/example/data/Data.kt

```diff
@@ -0,0 +1,59 @@
+package com.example.data
+
+import android.net.ConnectivityManager
+import android.util.Log
+import com.example.domain.Base
+import com.example.domain.Item
+import com.google.gson.GsonBuilder
+import retrofit2.*
+import retrofit2.converter.gson.GsonConverterFactory
+
+class Data {
+    var repo: MutableMap <String, MutableList<Base>> = mutableMapOf()
+
+    private fun loadData(item: String, callback: (List<Base>) -> (Unit)) {
+        var retrofit: Retrofit = Retrofit.Builder()
+            .baseUrl("https://raw.githubusercontent.com/karl-park/com.ninetynine.healthysalad/master/server/")
+            .addConverterFactory(GsonConverterFactory.create(GsonBuilder().create()))
+            .build()
+        var service: DataGetter = retrofit.create(DataGetter::class.java)
+        var call: Call<Item> = service.listItems(item)
+        call.enqueue(object : Callback<Item> {
+            override fun onFailure(call: Call<Item>, t: Throwable) {
+                Log.d("Called", t.toString())
+            }
+            override fun onResponse(call: Call<Item>, response: Response<Item>) {
+                response.body()?.body?.data?.base?.forEach { item -> Log.d("Called", item.name) }
+                onLoadSuccess(response, item, callback)
+            }
+        })
+    }
+
+    interface NetworkCallback {
+        fun onSuccess(item: List<Base>)
+    }
+    fun loadItem(item: String, callback : (List<Base>) -> (Unit)) {
+        val fileName = "$item.json"
+        loadData(fileName, callback)
+    }
+/*
+    fun loadAllItems(callback: (List<Base>) -> Unit) {
```

Comment:
It's good practice to remove unused / commented out code, but if we think we might need it soon then we can keep it :)